pkg/serverless/trace/trace.go

```diff
@@ -38,6 +39,15 @@ type LoadConfig struct {
 // httpURLMetaKey is the key of the span meta containing the HTTP URL
 const httpURLMetaKey = "http.url"
 
+// tcpRemoteHostMetaKey is the key of the span meta containing the TCP remote host i.e 127.0.0.1
+const tcpRemoteHostMetaKey = "tcp.remote.host"
+
+// tcpRemotePortMetaKey is the the key of the span meta containing the TCP remote port e.g 8124
+const tcpRemotePortMetaKey = "tcp.remote.port"
```

Comment:
I believe we'll also need to make sure we filter this out for `tcp.local.host` and `tcp.local.port`. From our previous conversation, it looks like this was remapped:
![image](https://user-images.githubusercontent.com/25290232/217277075-debd3e12-184d-4868-8ae9-6d5a52fedb35.png)

Reply:
I double-checked the span metadata that we get, and we somehow don't parse the tcp.local.port. An example - https://a.cl.ly/jkuOxzXb, https://a.cl.ly/v1uEZdQl. I believe the span metadata has tcp.remote.host, and tcp.remote.port irrespective of the span tags that we see in the UI.